project/tikv/project.json

```diff
@@ -0,0 +1,108 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+
+    "maintainers": [
+        "BusyJay",
+        "andylokandy",
+        "breeswish",
+        "hicqu",
+        "nrc",
+        "skyzh",
+        "yiwu-arbug"
+    ],
+
+    "committers": [
+        "5kbpers",
+        "AndreMouche",
+        "Connor1996",
+        "Little-Wallace",
+        "MyonKeminta",
+        "NingLin-P",
+        "TennyZhuang",
+        "bobotu",
+        "brson",
+        "cfzjywxk",
+        "coocood",
+        "disksing",
+        "gengliqi",
+        "hunterlxt",
+        "imtbkcat",
+        "iosmanthus",
+        "jackysp",
+        "lonng",
+        "lysu",
+        "niedhui",
+        "overvenus",
+        "sunxiaoguang",
+        "sticnarf",
+        "tabokie",
+        "tiancaiamao",
```

Comment:
Please remove me, I don't write or review rust code nowadays.

Reply:
Thanks for your report. I will add an emeritus committer field and move you there. An emeritus committer won't be requested or have effective permission on the repo, but we do respect your works on this project :)